05-onchain.md

```diff
@@ -185,231 +161,193 @@ Similarly, when node A sees a *commitment transaction* from B:
-because they were trimmed as dust, or in the case where B has two
-valid commitment transactions, and the HTLCs differ in each.
-
-The HTLC has *timed out* once the depth of the latest block is equal
+The HTLC output has *timed out* once the depth of the latest block is equal
 or greater than the HTLC `cltv_expiry`.
+The method by which the HTLC times out differs depending
+on which party the commitment transaction belongs to.
 
-The method by which we time out the HTLC output differs depending
-on whether it's our own commitment transaction, or theirs.
+There can be HTLCs that are not represented by an output: either
+because they were trimmed as dust or because the receipient has two
+valid commitment transactions, and the HTLCs differ in each.
 
 ## Requirements
 
-
 If the HTLC output is spent using the payment preimage, the HTLC
 output is considered *irrevocably resolved*, and the node MUST extract
 the payment preimage from the transaction input witness.
 
-
 If the HTLC output has *timed out* and not been *resolved*, the node
 MUST *resolve* the output and MUST fail the corresponding incoming
 HTLC (if any) once the resolving transaction has reached reasonable
-depth.  If the transaction is the node's own
+depth. If the transaction is the node's own
 commitment transaction, it MUST *resolve* the output by spending it
 using the HTLC-timeout transaction, and the HTLC-timeout
 transaction output MUST be *resolved* as described in "On-chain HTLC
-Transaction Handling".  Otherwise it MUST resolve the
+Transaction Handling". Otherwise it MUST resolve the
 output by spending it to a convenient address.
 
-
-For any committed HTLC which does not have an output in this
+For any committed HTLC that does not have an output in this
 commitment transaction, the node MUST fail the corresponding incoming
 HTLC (if any) once the commitment transaction has reached reasonable
 depth, and MAY fail it sooner if no valid commitment transaction
 contains an output corresponding to the HTLC.
 
-
 ## Rationale
 
-
-If the commitment transaction is theirs, the only way to spend the
+If the commitment transaction is the recipient's, the only way to spend the
 HTLC output using a payment preimage is for them to use the
-HTLC-success transaction.  If the commitment transaction is ours, they
+HTLC-success transaction. If the commitment transaction is the payer's, the recipient
 could create any transaction using the preimage.
 
-
-The payment preimage either serves to prove payment (if this node
-originated the payment), or to redeem the corresponding incoming HTLC
-from another peer.  Note that we don't care about the fate of the
-HTLC-spending transaction itself once we've extracted the payment
+The payment preimage either serves to prove payment (for the payer) or
+to redeem the corresponding incoming HTLC from another peer (for the
+recipient). Note that the parties don't care about the fate of the
```

Comment:
No.  This paragraph is differentiating between the two kinds of "payer" (in your nomenclature).  One is the origin of the payment, the other is a payer because they're also a recipient from a different channel (i.e. forwarding).

Reply:
Got it. (Resolved.)